DirectProgramming/Fortran/EdgeDetection/simple-binary-images/README.md

```diff
@@ -0,0 +1,121 @@
+# Simple Edge Detection Sample
+Segmentation is a common operation in image processing to find the boundaries of objects in an image.
+This sample implements a simple edge detection algorithm to find object boundaries in a binary image.
+However, this sample is more about offloading Fortran code to a GPU than it is about edge detection.
+The algorithm is implemented in two different but functionally equivalent ways. First, it is implemented
+using ordinary nested for-loops that are parallelized using OpenMP directives. Second, it is implemented
+using a single DO CONCURRENT loop, which is parallelized using the OpenMP backend. In either case, the
+Intel&reg; OpenMP runtime library is capable of offloading the edge detection loops to a GPU.
+
+| Optimized for       | Description
+|:---                 |:---
+| OS                  | Linux* Ubuntu* 18.04 or newer
+| Hardware            | Intel&reg; CPUs and GPUs
+| Software            | Intel&reg; Fortran Compiler
+| What you will learn | How to offload Fortran loops to a GPU
+| Time to complete    | 15 minutes
+
+## Purpose
+This sample demonstrates two Fortran implementations of edge detection:
+
+ 1. img_seg_omp_target.F90 implements edge detection on binary images using ordinary for-loops and OpenMP target directives
+ 2. img_seg_do_concurrent.F90 implements edge detection on binary images using only a DO CONCURRENT loop
+
+The implementations are functionally equivalent. In both cases, the OpenMP runtime library is used to parallelize the
+edge detection loops, regardless of whether they are run on the CPU or offloaded to a GPU.
+
+## Key Implementation Details
+[Using Fortran DO CONCURRENT for Accelerator Offload](https://www.intel.com/content/www/us/en/developer/articles/technical/using-fortran-do-current-for-accelerator-offload.html) provides more detailed descriptions of each example code, and discusses the relative merits of each approach.
+
+## License
```

Comment:
@hagabb Can you put the ``License`` section at the end?

Reply:
I moved the License section to the end of the README.